content/contribute/_index.en.md

```diff
@@ -1,15 +1,17 @@
 ---
-title: "Get involved"
+title: "Contribute"
```

Comment:
Why did you change this title again? Now the page will have the same title twice, once in the header and again as main title... This was the reason I originally modified it to `Get involved`

Reply:
@veroandreo sorry but I don't understand, now (with last changes) the page look like this 
![contribute](https://user-images.githubusercontent.com/178741/82630227-68bf9b80-9bf2-11ea-8dde-1f8af867b180.png)

It has the same structure of other landing pages